package.json

```diff
@@ -11,13 +11,14 @@
     "test": "npm run test:unit && npm run test:functional",
     "test:unit": "karma start --single-run",
     "test:unit:watch": "karma start",
-    "test:unit:saucelabs": "karma start karma-saucelabs.conf.js",
     "test:unit:coverage": "karma start --single-run --reporters spec,coverage",
     "test:functional": "rollup -c --environment BUILD:prodStandalone && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
     "test:functional:watch": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" BUILD=\"prodStandalone\" ./scripts/watchFunctionalTests.js",
-    "test:functional:ci": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe 'browserstack:ie@11:Windows 10'",
-    "test:functional:cron:prod": "ALLOY_ENV=\"prod\" testcafe chrome",
+    "test:functional:ci:ie": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe 'browserstack:ie@11:Windows 10'",
+    "test:functional:ci:chrome": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe 'browserstack:chrome:Windows 10'",
+    "test:functional:cron:prod": "ALLOY_ENV=\"prod\" testcafe 'browserstack:chrome:Windows 10'",
     "test:functional:api": "testcafe chrome:headless",
+    "sandbox:build": "cd sandbox && npm run build",
```

Comment:
Is this script something you're using manually to upload the sandbox to alloy.io? If so, is that upload process something we can add as a github action?

Reply:
Btw, you don't need to make a github action for this in this PR...I was just curious.

Reply:
The action I used to upload the sandbox to the VMs on Azure was before the firewall egress changes. I'll open a separate PR for the sandbox hosting since it requires some debugging.